src/hal/src/pm_stm32f4.c

```diff
@@ -407,16 +407,46 @@ void pmTask(void *param)
   }
 }
 
+/**
+ * Power management log variables.
+ */
 LOG_GROUP_START(pm)
+/**
+ * @brief Battery voltage [V]
+ */
 LOG_ADD(LOG_FLOAT, vbat, &batteryVoltage)
+/**
+ * @brief Battery voltage [mV]
+ */
 LOG_ADD(LOG_UINT16, vbatMV, &batteryVoltageMV)
+/**
+ * @brief BigQuad external voltage measurement [V]
+ */
 LOG_ADD(LOG_FLOAT, extVbat, &extBatteryVoltage)
+/**
+ * @brief BigQuad external voltage measurement [mV]
+ */
 LOG_ADD(LOG_UINT16, extVbatMV, &extBatteryVoltageMV)
+/**
+ * @brief BigQuad external current measurement [V]
+ */
 LOG_ADD(LOG_FLOAT, extCurr, &extBatteryCurrent)
+/**
+ * @brief Battery charge current [A]
+ */
 LOG_ADD(LOG_FLOAT, chargeCurrent, &pmSyslinkInfo.chargeCurrent)
+/**
+ * @brief PM state [battery=0, charging=1, charged=2, lowPower=3, shutDown=4]
```

Comment:
In other places we use a table for documentation, something like this?

```
@brief State of power management

State | Meaning
----------------------
0     | battery
1     | charging
2     | charged
3     | low power
4     | shutdown
```

Reply:
Fixed